service/app/models/user_account.rb

```diff
@@ -0,0 +1,9 @@
+class UserAccount < ActiveRecord::Base
+	include ApplicationModel
```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)